target/linux/ath79/image/generic.mk

```diff
@@ -438,6 +438,15 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/kuwfi_cpe2g-9531
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := KuWfi
+  DEVICE_MODEL := CPE2G-9531
+  DEVICE_VARIANT :=
```

Comment:
No need to put empty DEVICE_VARIANT here. Remove it.

Reply:
fixed